test/models/event_test.rb

```
@@ -0,0 +1,37 @@
+require "test_helper"
+
+class EventTest < ActiveSupport::TestCase
+  def setup
+    @user = User.create!(email: "event_test_#{Time.now.to_i}@example.com", display_name: "Event Tester", password: "password")
```

RuboCop (1.75.5)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)